src/features/verify/verifyHelper.ts

```diff
@@ -0,0 +1,13 @@
+import { window } from 'vscode'
+import { labels } from '../../messages/messages'
+
+const verifyInput = async () => {
+  const verifyInputBox = await window.showInputBox({
+    prompt: labels.verifyAVC,
+  })
+  return verifyInputBox
+}
```

Comment:
Shouldn't we ask user to select a JSON file here?

Reply:
Do you mean we should use a file picker and just be able to select a JSON file and thats it? Currently its set as an input and the use can paste some JSON into it and it will return the verification....

Reply:
Yeah I think in most cases user will have a JSON file (VC) to validate, so IMO it should be a file picker.